axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConnectionManager.java

```diff
@@ -336,33 +377,21 @@ private synchronized void startInstructionStream(String context,
                                                      ClientIdentification clientIdentification) {
         logger.debug("Start instruction stream to node [{}] for context [{}]", name, context);
         SynchronizedStreamObserver<PlatformInboundInstruction> inputStream = new SynchronizedStreamObserver<>(
-                PlatformServiceGrpc.newStub(intercepted(
-                        context, channels.get(context)
-                )).openStream(new UpstreamAwareStreamObserver<PlatformOutboundInstruction>() {
+                PlatformServiceGrpc.newStub(intercepted(context, channels.get(context)))
+                                   .openStream(new UpstreamAwareStreamObserver<PlatformOutboundInstruction>() {
 
                     @Override
                     public void onNext(PlatformOutboundInstruction messagePlatformOutboundInstruction) {
-                        handlers.getOrDefault(context, Collections.emptyList())
-                                .forEach(consumer -> consumer.accept(messagePlatformOutboundInstruction));
-
-                        switch (messagePlatformOutboundInstruction.getRequestCase()) {
-                            case NODE_NOTIFICATION:
-                                logger.debug("Received: {}", messagePlatformOutboundInstruction.getNodeNotification());
-                                break;
-                            case REQUEST_RECONNECT:
-                                Consumer<String> reconnect = (c) -> {
-                                    notifyConnectionChange(disconnectListeners, c);
-                                    getRequestStream().onCompleted();
-                                    scheduleReconnect(context, true);
-                                };
-                                for (Function<Consumer<String>, Consumer<String>> interceptor : reconnectInterceptors) {
-                                    reconnect = interceptor.apply(reconnect);
-                                }
-                                reconnect.accept(context);
-                                break;
-                            case REQUEST_NOT_SET:
-                                break;
-                        }
+                        Collection<BiConsumer<PlatformOutboundInstruction, StreamObserver<PlatformInboundInstruction>>> defaultHandlers = Collections
+                                .singleton((poi, stream) -> sendUnsuccessfulInstructionResult(
```

Comment:
change messagePlatformOutboundInstruction.getInstructionId() to poi.getInstructionId() and it can be a constant.